packages/story-editor/src/components/colorPicker/useDeleteColor.js

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { useCallback } from '@web-stories-wp/react';
+/**
+ * Internal dependencies
+ */
+import { useStory } from '../../app';
+
+function useDeleteColor({ setIsEditMode }) {
```

Comment:
I feel this would be a bit cleaner and independent if this callback was just `onEmpty` that would be invoked if there were no more saved colors. Then it isn't mixed up with the notion of edit mode, which is otherwise irrelevant for this hook. And it would just be from the outside:

```js
useDeleteColor({ onEmpty: () => setIsEditMode(false) });
```

(but probably memoized)